src/@types/stores.types.ts

```diff
@@ -87,6 +87,8 @@ export interface AppStore extends TypedStore {
   authRequestFailed: () => void;
   autoLaunchOnStart: () => void;
   automaticUpdates: boolean;
+  twoFactorAutoCatcher: boolean;
```

Comment:
Looking through the code, this seems like a toggle? If so, please differentiate between the toggle and the value via more appropriate names. For eg, the toggle can be `twoFactorAutoCatcherFlag` and the other variable could be without the `Array` suffix.

Reply:
what about `enableTwoFactorAutoCatcher` or `isTwoFactorAutoCatcherEnabled`? 
the other toogle's don't have straight naming concept which we can follow :)
i prefer `isTwoFactorAutoCatcherEnabled` and the array `twoFactorAutoCatcherMatcher`.

Reply:
pls use `isTwoFactorAutoCatcherEnabled` - I actually noticed the same in 2 other feature-enablement flags, and have a PR open (waiting for approval) for exactly this reason
the 2nd variable (I dont think the datatype is an array, its more of a comma-separated string), can be `twoFactorAutoCatcherMatcher`

Reply:
done